src/main/resources/static/js/subjectFormModal.js

```diff
@@ -0,0 +1,3 @@
+$(document).ready(function() {
+    $("#modal-placeholder").load("/userSubject/form");
```

Comment:
decide on weather you want to use kebab-case or camelCase for the ids but do not mix everywhere